pyatv/support/facade.py

```diff
@@ -432,6 +432,9 @@ def service(self) -> interface.BaseService:
 
         raise RuntimeError("no service (bug)")
 
+    def get_service(self, protocol: Protocol) -> interface.BaseService:
```

Comment:
This should not be here (it would also have to be added to `interface.AppleTV` if it were). See my comment in the test on what to do.